CMakeLists.txt

```diff
@@ -213,6 +213,11 @@ if(CMAKE_C_COMPILER_ID MATCHES "MSVC")
   string(APPEND CMAKE_EXE_LINKER_FLAGS " /DYNAMICBASE:NO")
   string(APPEND CMAKE_EXE_LINKER_FLAGS " /FIXED")
 
+  # Enforce some behaviors as standards-conformant when they don't default as such
+  add_compile_options(/Zc:strictStrings /Zc:rvalueCast)
```

Comment:
One option per line please. Since those are a bit obscure, a short comment explaining what they do and why we want them would be nice too.